website/__tests__/report-issue.test.js

```diff
@@ -0,0 +1,33 @@
+import Vue from "vue";
+import { render, screen } from "@testing-library/vue";
+// eslint-disable-next-line import/no-unresolved
+import report from "@/pages/report.vue";
+
+describe("/report", () => {
+  it("renders input fields with data populated", async () => {
+    render(report, {
+      stubs: {
+        "b-switch": true,
+        "b-button": true,
+        "b-input": "<input />",
+        "b-field": Vue.component("b-field", {
+          props: ["label"],
+          template: "<label role='label'>{{label}}<slot /></label>",
+        }),
+      },
+    });
+
+    // RAM
+    expect(screen.getByDisplayValue("undefined GB")).toBeTruthy();
```

Comment:
this looks weird, but it's because the virtual test environment doesn't have access to information like this. In a real browser it say "16 GB" etc.